lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1963,9 +1962,10 @@ definitions:
     minLength: 1
     required:
       - cell_id
-      - device_class
-      - transmit_enabled
     properties:
+      ip_address:
+        type: string
+        format: ipv4
```

Comment:
properties that aren't relevant to managed radios shouldn't be in this struct. you should actually change the configuration type to a one-of and specify separate concrete types for managed and unmanaged radios